mk/check.mk

```diff
@@ -2,7 +2,7 @@
 fmt: fmt/go fmt/proto ## Dev: Run various format tools
 
 .PHONY: fmt/go
-fmt/go: ## Dev: Run go fmt
+fmt/go: golangci-lint-fmt ## Dev: Run go fmt
 	go fmt ./...
```

Comment:
is this still needed then?

Reply:
Do you mean `go fmt ./...`?

Reply:
yes, if `golangci-lint` is running `fmt` do we still need it?

Reply:
removed